apps/test/unit/lib/tools/jsdebugger/reduxTest.js

```diff
@@ -0,0 +1,264 @@
+import sinon from 'sinon';
+import {expect} from '../../../../util/configuredChai';
+import {getStore, registerReducers, stubRedux, restoreRedux} from '@cdo/apps/redux';
+import {reducers, selectors, actions} from '@cdo/apps/lib/tools/jsdebugger/redux';
+import CommandHistory from '@cdo/apps/lib/tools/jsdebugger/CommandHistory';
+import {singleton as studioApp} from '@cdo/apps/StudioApp';
+import ObservableEvent from '@cdo/apps/ObservableEvent';
+import JSInterpreter from '@cdo/apps/JSInterpreter';
+
+class StubInterpreter {
```

Comment:
Wait.. @joshlory uses the real interpreter [in unit tests over here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/JSInterpreterTest.js) and it doesn't look like ace editor caused any problems.